src/content/Angular-CSharp/ClientApp/src/app/fetch-data/fetch-data.component.ts

```diff
@@ -8,8 +7,15 @@ import { HttpClient } from '@angular/common/http';
 export class FetchDataComponent {
   public forecasts: WeatherForecast[] = [];
 
-  constructor(http: HttpClient, @Inject('BASE_URL') baseUrl: string) {
-    http.get<WeatherForecast[]>(baseUrl + 'weatherforecast').subscribe(result => {
+  constructor(@Inject('BASE_URL') baseUrl: string) {
```

Comment:
Note: I removed all usages of `@angular/common/http` in favor of `fetch` for simplicity.

Reply:
We should keep the HttpClient. It's the idiomatic way to do things in angular and plugs in with the change detection mechanisms. https://angular.io/guide/http